code/modules/paperwork/records/info/zayin.dm

```diff
@@ -127,3 +127,9 @@
 		"After work was completed, the fireflies glowed brighter.",
 		"The fireflies increased healing of HP and SP as long as the employee stayed in the room.",
 		"While the fireflies were glowing, no other work could be completed on O-02-183.")
+//Bead Marker
+/obj/item/paper/fluff/info/zayin/bead_marker
+	abno_type = /mob/living/simple_animal/hostile/abnormality/bead_marker
+	abno_code = "D-04-451"
```

Comment:
Do you happen to want to donate?

Reply:
> Do you happen to want to donate?

I mean, I _could,_ but how does that relate to the PR?